web/modules/custom/custom_page/custom_page.behat.inc

```diff
@@ -74,4 +78,97 @@ class CustomPageSubContext extends DrupalSubContextBase implements DrupalSubCont
     $this->visitPath($node->url());
   }
 
+  /**
+   * Checks the number of custom pages a collection or solution has.
+   *
+   * @param int $count
+   *   The expected number of custom pages.
+   * @param string $name
+   *   The name of the collection or solution to which the custom pages belong.
+   * @param string $type
+   *   The RDF entity type to which the custom pages belong. Can be either
+   *   'collection' or 'solution'.
+   *
+   * @Then the :name :type should have :number custom page(s)
+   */
+  public function assertContactInformationCount($count, $name, $type) {
```

Comment:
Either this should be $number instead of count, or the step definition should change to count.